staging/kos/.gitignore

```diff
@@ -1,14 +1,17 @@
 ## Exclusions for KOS
 
 # The built binaries
-images/*/*r
-images/*/*/*[rt]
+images/network-apiserver/network-apiserver
+images/controllers/ipam/ipam-controller
+images/controllers/connectionagent/connection-agent
 images/controllers/connectionagent/test-scripts
+images/controllers/subnet/validator/subnets-validator
```

Comment:
or how about everything that does not end with "e"?

Reply:
I would leave that as it is for now. I'm working separately on refactoring the IPAM and the subnets validation controller, which requires changing the directory hierarchy you referenced in this comment, hence the .gitignore will need to change as well. Overall, I think the way our controllers are organized in packages, the package names and and file names could be made more consistent, so I'll probably change that in the PR with the refactoring of the IPAM controller and subnets validation controller start up (which I'm yet to open).